### DISCUSSION TITLE
Pro Time Picker Hide Event

### DISCUSSION THREAD

**Discussion by toomanylogins** [coreui/coreui#829] [General]
Is the time picker hide event triggered when the ok or cancel buttons are clicked for the time picker ?

`    endTime.addEventListener('hide.coreui.time-picker', (timeString, localeTimeString, date) => {
        $('#duration').attr('data-coreui-time', myTimer.getTimeValues().toString());
    const duration = document.getElementById('duration');
        coreui.TimePicker.getInstance(duration).update();

    });
`

I cannot seem to trigger the event ? The timeChange.coreui.time-picker is working but trying to get the event on OK or cancel

**Comment by toomanylogins** [comment]
Resolved this by adding custom class to confirm button and getting click event. Still not sure why hide event not working though.

**Reply by mrholek** [reply]
The `hide` event is fired when you click the cancel or ok button. Can you show me your complete code?
